cluster-autoscaler/cloudprovider/gce/autoscaling_gke_client.go

```diff
@@ -24,7 +24,7 @@ import (
 
 // This flag is outside main as it's only useful for test/development.
 var (
-	gkeAPIEndpoint = flag.String("gke-api-endpoint", "", "GKE API endpoint address. This flag is used by developers only. Users shouldn't change this flag.")
+	GkeAPIEndpoint = flag.String("gke-api-endpoint", "", "GKE API endpoint address. This flag is used by developers only. Users shouldn't change this flag.")
```

Comment:
Will be private again once it's removed from GCE. For now, it has to be available in both packages since flag can't be redeclared.